Makefile

```diff
@@ -37,6 +37,10 @@ welcome:
 	@echo "Or ask questions on our mailing list:"
 	@echo "    users@riot-os.org (http://lists.riot-os.org/mailman/listinfo/users)"
 
+.PHONY: print-versions
```

Comment:
Why not add this to the `.PHONY` targets above in line 3?

Reply:
Ok, I changed that :)